pkg/server/container_create_windows.go

```diff
@@ -354,7 +456,7 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 					g.SetLinuxRootPropagation("rslave")
 				}
 			default:
-				log.G(ctx).Warnf("Unknown propagation mode %v for hostPath %q, defaulting to rprivate", m.GetPropagation(), src)
+				log.G(context.TODO()).Warnf("Unknown propagation mode %v for hostPath %q, defaulting to rprivate", m.GetPropagation(), src)
```

Comment:
I feel like we should just pass in ctx to this function. Was there a reason you did this instead?

Reply:
no particular reason, I can add this.